src/core/types/Chains.ts

```diff
@@ -0,0 +1,8 @@
+export const SUPPORTED_CHAIN_IDS = [
+  1,
+  10,
+  137,
+  8453,
```

Comment:
it would be more clear if we have chain name here or chain object from viem. hard to know what chain represent by `8453` at a glance.

Reply:
Comments?